### DISCUSSION TITLE
How to setup specific placement for Named TextField

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#8651] [Q&A] [answered]
Can i setup placement for specific field?

```
{
  "TextField": ["CustomerName" ],
  "contentType": [ "Customers",  ]
}
```
Something like this?!

**Comment by Skrypt** [comment]
I think you will need to use a differenciator in your placement like explained here : 

https://docs.orchardcore.net/en/dev/docs/reference/core/Placement/#placing-fields

**Reply by vitalybrandes** [reply]
I would like to arrange my content submit in Admin page moving some of fields to different tab.
From this url, unfortunately i didnt understand how to do that :-(

I have text field Customer name and text field customer id and i am wondering how to move only 2 this specific text fields to additional new tab "Customer".

**Comment by ns8482e** [comment, marked as answer]
Use differentiator as following in placement. This will move text filed to "Settings" tab in edit content screen

```
"TextField_Edit": [
  {
    "place": "Parts#Settings:0",
    "differentiator": "MyType-MyField"
  }
]
```

**Reply by vitalybrandes** [reply]
Perfecto!!!!

Thank you!
